app/forms/password_form.py

```diff
@@ -0,0 +1,12 @@
+from flask_login import current_user
+from flask.ext.wtf import Form
+from wtforms import StringField, PasswordField, SubmitField
+from flask_wtf.html5 import EmailField
+from wtforms.validators import ValidationError, DataRequired, EqualTo
+from app.models import User
+
+class PasswordForm(Form):
+    currentpassword = PasswordField('Current Password', render_kw={'class': 'form-control', 'placeholder': 'enter current password', 'required': True}, validators=[DataRequired('Please enter your current password.')])
```

Comment:
Normally, if you change your password you have to enter your current password one time (to confirm knowledge of the password) and the new password two times (to avoid having a typo and not knowing your new password after you changed it).

- [ ] `confirmpassword` should be after `newpassword` and equal to `newpassword`